src/status_im/contexts/settings/privacy_and_security/view.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im.contexts.settings.privacy-and-security.view
+  (:require
+    [quo.core :as quo]
+    [quo.theme]
+    [react-native.safe-area :as safe-area]
+    [status-im.contexts.settings.privacy-and-security.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn navigate-back
+  []
+  (rf/dispatch [:navigate-back]))
```

Comment:
Hi @ilmotta thank you for the PR.

Should we create a new namespace, something like `navigation/utils` or `utils/global-function` and move this function there? Related issue https://github.com/status-im/status-mobile/issues/19971

(Not blocker for the PR)

Reply:
I would like to do that @Parveshdhull. I'll remember that for the follow-up 👍🏼